dev-client/src/auth/index.ts

```diff
@@ -78,7 +82,11 @@ async function exchangeToken(
 ) {
   const payload = await request<AuthTokens>({
     path: '/auth/token-exchange',
-    body: {provider, jwt: identityJwt},
+    body: {
+      provider,
+      client_id: Constants.expoConfig!.ios?.bundleIdentifier,
```

Comment:
Suggestion/question: Could this use the `APP_CONFIG.appleClientId` from config/index.ts? Right now it seems like the logic of "use the bundleIdentifier as the client id" is happening in two places and am wondering if there's a way to have one place that's the source of truth for the client id